ch09/ex-create-element.md

```diff
@@ -136,3 +136,57 @@ by using `insertBefore` with a reference node of null.
                TextLayout(x=13, y=35.25, width=48, height=12, word=More)
                TextLayout(x=73, y=35.25, width=84, height=12, word=content)
                InputLayout(x=169, y=35.25, width=200, height=12, node=<input>)
+
+
+
+Test: Screen redraws after
+
+Test `appendChild`
+
+    >>> script = """
+    ... new_elt = document.createElement("input");
+    ... my_p = document.querySelectorAll('p')[0];
+    ... my_p.appendChild(new_elt);
+    ... """
+    >>> html_url = wbemocks.socket.serve("<body><div>Some content</div><p>More content</p></body>")
+    >>> this_browser.new_tab(browser.URL(html_url))
+    >>> this_browser.active_tab.js.run(script)
+
+    >>> print(browser.print_tree(this_browser.active_tab.document))
+     DocumentLayout()
+       BlockLayout(x=13, y=18, width=774, height=30.0, node=<html>)
+         BlockLayout(x=13, y=18, width=774, height=30.0, node=<body>)
+           BlockLayout(x=13, y=18, width=774, height=15.0, node=<div>)
+             LineLayout(x=13, y=18, width=774, height=15.0)
+               TextLayout(x=13, y=20.25, width=48, height=12, word=Some)
+               TextLayout(x=73, y=20.25, width=84, height=12, word=content)
```

Comment:
Uhh, I don't see the new `<input>` element here? I think this was a bug in my instructor solution!